ClientKit/ClientKit.cs

```diff
@@ -80,6 +80,9 @@ public class ClientContext : IDisposable
             [DllImport(OSVRCoreDll, CallingConvention = CallingConvention.Cdecl)]
             public extern static Byte osvrClientGetStringParameter(SafeClientContextHandle ctx, string path, StringBuilder buf, UIntPtr len);
 
+            [DllImport(OSVRCoreDll, CallingConvention = CallingConvention.Cdecl)]
+            internal extern static Byte osvrClientCheckStatus(SafeClientContextHandle ctx);

```

Comment:
They will all be internal eventually. Didn't want to add new public P/Invokes that I'd have to ```[Obsolete]``` later.